examples/object_detector/yolo/yolov8/custom_handler.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+from collections import Counter
+
+import torch
+from torchvision import transforms
+from ultralytics import YOLO
+
+from ts.torch_handler.object_detector import ObjectDetector
+
+logger = logging.getLogger(__name__)
+
+try:
+    import torch_xla.core.xla_model as xm
+
+    XLA_AVAILABLE = True
+except ImportError as error:
+    XLA_AVAILABLE = False
+
+
+class Yolov8Handler(ObjectDetector):
+    image_processing = transforms.Compose(
+        [transforms.Resize(640), transforms.CenterCrop(640), transforms.ToTensor()]
+    )
+
+    def __init__(self):
+        super(Yolov8Handler, self).__init__()
+
+    def initialize(self, context):
```

Comment:
I think the value of self.device and self.model_pt_path is covered by basehandler.

Reply:
@lxning We need to change the design of the basehandler `initialize` method for this to work.  Then we can use `super`. This is something I discussed with @msaroufim .   Currently, this doesn't work because it fails here for this model https://github.com/pytorch/serve/blob/examples/yolov8/ts/torch_handler/base_handler.py#L167.

Reply:
We need to move parts of the code into a `try` , `except` block.

Reply:
I'm still not sure I entirely follow how `try/except` will help in this case - but if the model is failing on `eval()` that seems off? What's the error?

Reply:
Using various try except blocks would ensure many of the properties are set using super. Ex: self.device,self.map_location, self.manifest etc. This could have also avoided https://github.com/pytorch/serve/issues/2480

Ultralytics yolov8 has their own method to load the model and this needs to be called in our custom load method. Its just one line. 
https://github.com/ultralytics/ultralytics